failed to build kf5-kjs 5.8, require KF5DocTools

```
/private/tmp/kf5-kjs-H04jfN/kjs-5.8.0$ cmake . -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/kf5-kjs/5.8.0 -DCMAKE_BUILD_TYPE=None -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DCMAKE_CXX_FLAGS='-D_DARWIN_C_SOURCE'
CMake Error at CMakeLists.txt:48 (find_package):
  By not providing "FindKF5DocTools.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "KF5DocTools", but CMake did not find one.

  Could not find a package configuration file provided by "KF5DocTools" with
  any of the following names:

    KF5DocToolsConfig.cmake
    kf5doctools-config.cmake

  Add the installation prefix of "KF5DocTools" to CMAKE_PREFIX_PATH or set
  "KF5DocTools_DIR" to a directory containing one of the above files.  If
  "KF5DocTools" provides a separate development package or SDK, be sure it
  has been installed.


-- Configuring incomplete, errors occurred!
See also "/tmp/kf5-kjs-H04jfN/kjs-5.8.0/CMakeFiles/CMakeOutput.log".
See also "/tmp/kf5-kjs-H04jfN/kjs-5.8.0/CMakeFiles/CMakeError.log".
```

--- a/kf5-kjs.rb
+++ b/kf5-kjs.rb
@@ -10,6 +10,7 @@ class Kf5Kjs < Formula
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "haraldf/kf5/kf5-extra-cmake-modules" => :build
+  depends_on "haraldf/kf5/kf5-kdoctools" => :build
   depends_on "qt5" => "with-d-bus"
   depends_on "pcre"
 

